pig/src/main/java/org/apache/iceberg/pig/IcebergPigInputFormat.java

```diff
@@ -29,7 +29,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import org.apache.commons.lang.SerializationUtils;
+import org.apache.commons.lang3.SerializationUtils;
```

Comment:
Why did we update the to this api?

Reply:
`lang` is being considered as deprecated, and the linter complaints that we should move to `lang3`.